### PR TITLE
Product version any user

### PR DESCRIPTION
**Pull request by KevinFCormier**
https://github.com/stolostron/backlog/issues/20234

### CODE REVIEW

File: backend/src/routes/events.ts [hunk 1]
```diff
@@ -185,7 +185,7 @@ async function listAndWatch(options: IWatchOptions) {
 }
 
 async function listKubernetesObjects(options: IWatchOptions) {
-    const serviceAcccountToken = getServiceAcccountToken()
+    const serviceAcccountToken = getServiceAccountToken()
```
Comment (mark-nc) [inline backend/src/routes/events.ts:188]: The const var name still has the extra "c" `const serviceAcccountToken`

File: backend/src/routes/events.ts [hunk 2]
```diff
@@ -253,7 +253,7 @@ async function listKubernetesObjects(options: IWatchOptions) {
 }
 
 async function watchKubernetesObjects(options: IWatchOptions, resourceVersion: string) {
-    const serviceAcccountToken = getServiceAcccountToken()
+    const serviceAcccountToken = getServiceAccountToken()
```
Comment (mark-nc) [inline backend/src/routes/events.ts:256]: The const var name still has the extra "c" `const serviceAcccountToken`

**Review by mark-nc — CHANGES_REQUESTED** [review]
Just two comments in backend/src/routes/events.ts but the rest looks good.

File: backend/src/routes/events.ts (hunk 1 above)
Comment (KevinFCormier) [reply backend/src/routes/events.ts]: Fixed

File: backend/src/routes/events.ts (hunk 2 above)
Comment (KevinFCormier) [reply backend/src/routes/events.ts]: Fixed

**Review by mark-nc — APPROVED** [review]
/lgtm

**Pull request merged**